ci/ci/build.py

```diff
@@ -93,7 +94,7 @@ def request(step):
 
             for s in requested_steps:
                 request(s)
```

Comment:
Rather than constructing requested_steps, you could deforest and do:

```
  for step_name in requested_step_names:
    request(name_step[step_name])
```